api/time_bucket_ng.md

```diff
@@ -54,8 +54,10 @@ timezone and so the function can't be used with continuous aggregates.
 The function returns the bucket's start time. The return value type is the
 same as `ts`.
 
+>>>>>>> latest
 ### Sample Usage
 
+>>>>>>> origin/latest
```

Comment:
Not sure how these have crept in, it's from resolving a merge conflict, and they need to be taken out.